### ISSUE TITLE
Feature request: per vertex color support

### ISSUE THREAD

**Issue by cdsousa** [JuliaIO/MeshIO.jl#57]
As I understand, GeometryBasics.jl supports color metadata on meshes. It would be very good if MeshIO.jl knew how to import color from formats that support it. With new GeometryBasics.jl metadata system it wouldn't even need to understand color representations, it could just forward it as metadata.
Does this makes sense at all?

**Comment by SimonDanisch** [comment]
Yes, it's just a matter of parsing it and adding the parses arreay as `color = parsed_colors` to the mesh ;)

**Comment by lazarusA** [comment]
@SimonDanisch  do you have an example? somewhere. I'm looking into loading a mesh with a texture, so far, no clues.
